src/integration_tests/schemas/e2e_schema_integration_test.py

```diff
@@ -59,6 +59,24 @@ def test_schema_e2e(self):
         response_as_json = test_schema_get_response.json()
         assert len(response_as_json) > 0
 
+        for survey_id in test_list_survey_id:
+            schema_post_response = session.post(
```

Comment:
Overwriting schema_post_response in each iteration will only keep the last response